ucp/_libs/ucx_api_dep.pxd

```diff
@@ -127,9 +126,11 @@ cdef extern from "ucp/api/ucp.h":
                                    ucp_worker_h *worker_p)
     void ucp_worker_destroy(ucp_worker_h worker)
 
-    ctypedef struct ucp_listener_h:
+    ctypedef struct ucp_listener:
         pass
 
+    ctypedef ucp_listener* ucp_listener_h
+
```

Comment:
Taken from [this UCX definition]( https://github.com/openucx/ucx/blob/8b6641b02ef5d217323f719d54dfb2e9142dad59/src/ucp/api/ucp_def.h#L168-L175 ).